### PR TITLE
Add 'DIRB'

### PR DESCRIPTION
**Pull request by khabib97**
- signed-off-by: khabib97 <kawser.habib.dev@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1283,6 +1283,8 @@ CyberTalks</b></a> - talks, interviews, and article about cybersecurity.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://github.com/gentilkiwi/mimikatz"><b>mimikatz</b></a> - a little tool to play with Windows security.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://github.com/sherlock-project/sherlock"><b>sherlock</b></a> - hunt down social media accounts by username across social networks.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://owasp.org/www-project-threat-dragon/"><b>OWASP Threat Dragon</b></a> - is a tool used to create threat model diagrams and to record possible threats.<br>
+&nbsp;&nbsp;:small_orange_diamond: <a href="https://tools.kali.org/web-applications/dirb"><b>DIRB</b></a> - is a Web Content Scanner. It looks for existing (and/or hidden) Web
+Objects. It basically works by launching a dictionary based attack against a web server and analizing the response.<br>
```
Comment (lodenrogue) [inline README.md:1287]: I would remove the line explaining the implementation details as the what is more important than the how in this project:

```
It basically works by launching a dictionary based attack against a web server and analizing the response.
```

**Review by lodenrogue — CHANGES_REQUESTED** [review]

**Review by khabib97 — COMMENTED** [review]
It seems, your suggested change is ok

**Comment by lodenrogue** [comment]
> It seems ok

No

**Comment by khabib97** [comment]
> > It seems ok
> 
> No

I told new change is ok. Sorry for the misunderstanding.

**Comment by lodenrogue** [comment]
I'm not seeing the change though

File: README.md (hunk 1 above)
Comment (khabib97) [reply README.md]: Removed

**Review by lodenrogue — APPROVED** [review]

**Review by lodenrogue — APPROVED** [review]

**Comment by lodenrogue** [comment]
Why was this closed?

**Pull request closed without merging**